app/views/users/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<ul>
+  <% @user.each do |user| %>
+    <li>
+      <%= user.id %>
+    </li>
```

Comment:
this isn't working for some reason.  Undefined method for user but everything looks good to me.  Can't figure it out.  Controller looks good.